stable/prometheus/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: prometheus
-version: 9.5.0
+version: 9.5.1
```

Comment:
I don't know, why Commit 84a48c7 from nicholascapo is part of this merge request. Meanwhile, I tried to bump Prometheus chart version because I got lint errors from one of the bots. But this wasn't helpful and I already reamoved this change. Hoernestly I don't know, what I can do at this point since I don't see any change of prometheus/Chart.yaml that I own.

Reply:
can you try the following?

```bash
git remote add btrappe https://github.com/btrappe/charts.git
git remote add upstream https://github.com/helm/charts.git
git fetch upstream
git pull upstream master --rebase
git push btrappe master -f
```

Reply:
About the reason of my changes:
I tried to test HA of the redis cluster and the sentinel-conform functionality of our services that uses the redis/sentinel cluster. To perform failover functionality as in real time, I just deleted slave or the master pod and found some abnormalities.

1. sentinel on master pod doesn't startup, if only setinel container on master pod is restarted (commit 84a48c7)

Interestingly, the redis chart creates a non-symmetrically set of pods: There is one "master" pod that own initially the redis master node and (optionally) several "slave" pods that owns initially a set of redis (and sentinel) nodes in slave mode. As long as the master node is kept untouched, scaling up/down slave pods and even recreating of suddently deleted slave pods work as intended (at least at first glance), because new slave pods can contact a redis via master service. Only if the master pod is created, it has the problem to determine, if it is the first pod of a new cluster (initial cluster startup) or if there are some remains of a running cluster, that lost its master pod. To address this problem, sameersbn added in commit 1adb31c (Chart version 8.0.16) some line that try to ask some sentinel for its list of sentinel IPs and add them directly to sentinel.conf. That works perfectly at pod fresh creation (non existing sentinel.conf and thus start with a copy of some template sentinel.conf) but lead to a persisting "error in sentinal configuration at line 21" (line number may vary), if the sentinal container is restarted only - you can easily simulate this by attaching to the sentinel container running on the pod of the master service and kill the sentinel process by typing "kill 1" - K8s will immediately restart the pod in-place (without wiping the folders) and thus, the code from sameersbn will append the well-known setinals a second time (there are still in sentinel.conf from the formerly running sentinel process) and lead to duplicate lines that makes the sentinel process to issue the configuration error instead of starting up. One solution would be to start always with a fresh sentinal.conf but it is worth to preserve the sentinel.conf whenever possible as it contains always the most current state of all sentinel cluster nodes. That's why I changed the code of sameersbn in that way that it is executed only if there is no former well-known sentinel node list found in sentinel.conf.

Reply:
2. reset sentinel list after helm upgrade
While I dealt with the problems of 1 and the article https://redis.io/topics/sentinel, I hit on the notice "Sentinels never forget already seen Sentinels". In an statically configured environment this has no implications if nodes will be restarted/recreated having the  former IP addresses. on k8s however, every pod (even after in-place updates or partial container restarts) will get always a new IP address. This leads the known sentinel list in sentinel.conf become longer and longer. Worse, at some point, the K8s cluster will recycle unused IP addresses and if some foreign redis/sentinal cluster gets such an IP address, two independent clusters will (at least partially) unify and lead to unexpected result. I observed such a problem on our cluster running several environments/stages of our mircoservice architecture and could not explain it at that time.

Reply:
3. reset sentinel after helm upgrade (Commit 69c4411)
Finnaly, the IP address problem decribed in 2. also ocurs while upgrading the helm chart, because pods are potentially are rebuild. Since it it is not ensured, that the master service pod is recreated and the whole upgrade doesn't take more than 120s, I added a job that is a hook of helm chart install/upgrade event and executes the same procedure as mentioned in 2. (reset all sentinals)

Reply:
@sameersbn - Thanks, that helped !